src/lib/transition_frontier_controller_tests/stubs.ml

```diff
@@ -624,42 +626,65 @@ struct
     type peer_config =
       {num_breadcrumbs: int; accounts: (Private_key.t option * Account.t) list}
 
-    type peer = {address: Network_peer.Peer.t; frontier: Transition_frontier.t}
+    type peer_with_frontier =
+      {peer: Network_peer.Peer.t; frontier: Transition_frontier.t}
```

Comment:
For the field `peer`, can it be of type `Unix.Inet_address`?

Reply:
We use this type to build both the network `ip_table` and the `peers` set, so it has to be a full `Peer.t`.

It might be better to pass just the `peers` to `Network.create`, and create the `ip_table` there, but that's just an optimization, and not worth doing for test code.